specification/cognitiveservices/data-plane/CustomVision/Prediction/stable/v3.0/Prediction.json

```diff
@@ -0,0 +1,917 @@
+﻿{
+  "swagger": "2.0",
+  "info": {
+    "version": "3.0",
+    "title": "Custom Vision Prediction Client"
+  },
+  "host": "southcentralus.api.cognitive.microsoft.com",
+  "basePath": "/customvision/v3.0/prediction",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/{projectId}/classify/models/{publishedModelName}/url": {
+      "post": {
+        "tags": [
+          "ImagePredictionApi"
+        ],
+        "summary": "Classify an image url and saves the result.",
+        "operationId": "ClassifyImageUrl",
+        "consumes": [
+          "application/json",
+          "text/json",
```

Comment:
Does this service really consume/accept all of these application types? text/json is not a [registered MIME type](https://www.iana.org/assignments/media-types/media-types.xhtml).

Reply:
This is autogenerated - will double check and remove invalid ones.